pages/_app.tsx

```diff
@@ -65,11 +65,14 @@ const segmentDestinationInterface: CustomDestination = {
 };
 
 const mixpanelDestinationInterface: CustomDestination = {
-  make: (env, apiKey) =>
+  make: (env, apiKey) => {
+    // eslint-disable-next-line
+    console.log('mixpanelDestinationInterface.make', env, apiKey);
```

Comment:
Does this fix the build?

Reply:
Haha no. Adding the staging key to Avo does it. Apparently Mixpanel crashes if it get's `null` as an api key